pkgs/development/interpreters/ruby/build-ruby-gem/gem-post-build.rb

```diff
@@ -64,8 +64,7 @@
 # this file is here to facilitate running it.
 #
 
-gem_path = ENV["GEM_PATH"]
-ENV["GEM_PATH"] = "\#{gem_path}\#{":" unless gem_path.nil? || gem_path.empty?}#{(gem_path+[gem_home]).join(":")}"
+Gem.use_paths "#{gem_home}", #{gem_path.to_s}
```

Comment:
However, as the code stands, `String#split` should always result in an Array of zero or more elements, so I don't think this bugged at the moment. Perhaps we should still touch this up a bit, though, to be a bit more robust in the face of future changes.